apps/comps/components/social-link-card.tsx

```diff
@@ -23,22 +23,28 @@ export const SocialLinkCard: React.FC<SocialLinkCardProps> = ({
         href={socialLink.url}
         target="_blank"
         rel="noopener noreferrer"
-        className={`bg-${socialLink.bg || "card"} hover:bg-card flex h-36 flex-col justify-between rounded-md p-6`}
+        className={`bg-${socialLink.bg || "card"} flex h-16 flex-col justify-between rounded-md p-4 transition-all duration-200`}
+        onMouseEnter={(e) => {
```

Comment:
Annoyingly, using pure tailwind hover classes works beautifully in chrome, but doesn't work at all in safari. So I had to resort to this onMouseEnter bs!

Reply:
weird, just tested replacing the onMouseEnter/Leave callbacks with `hover:brightness-80` on Safari 18.6 and it worked fine

Reply:
Wait, for real?!

Reply:
Ok, so weird. Now it's working for me?!